.github/actions/f3d-superbuild/action.yml

```diff
@@ -74,6 +74,7 @@ runs:
       echo VTK_COMMIT_SHA=`cat ./vtk_commit_sha` >> $GITHUB_ENV
 
   # OCCT coloring is disabled on macOS because of f3d-app/f3d#792
+  # OpenVDB is disabled on macOS arm64 because boost iostreams does not supports it yet
```

Comment:
Is it an issue in common-superbuild or Boost itself?

Reply:
the boost project in common-superbuild does not support it yet, i will clarify.